book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = [] # Initialize rentals as an empty array
+  end
```

Comment:
- [ ] Kindly add an association between a book and a person renting a book. A add rental method should be created to account for a book, the person renting, and the date it has/was rented. [Here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) is a link to the reason if you want to refresh.